### DISCUSSION TITLE
Npx serve .

### DISCUSSION THREAD

**Discussion by mminkowicz** [community/community#141107] [Programming Help]
### Select Topic Area

Question

### Body

For some reason when i add in terminal npx serve .
it will opne the localhost serve but wont show changes to the html document i made.

**Comment by Sofines** [comment]
If you're using npx to serve your project, it might not support automatic live reloading, so refresh the page each time you add changes or use a tool like "live-server"
npm install -g live-server
live-server

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @mminkowicz ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
